docs/ref/contrib/sitemaps.txt

```diff
@@ -345,7 +346,8 @@ Here's an example of a :doc:`URLconf </topics/http/urls>` using
     from django.contrib.sitemaps import GenericSitemap
     from django.contrib.sitemaps.views import sitemap
     from django.urls import path
-    from blog.models import Entry
+
+    from .models import Entry
```

Comment:
Probably this file should be left alone since sitemaps are likely to be in the root URLConf (see [djangoproject.com](https://github.com/django/djangoproject.com/blob/4f9acf7c8735db92d279fa7b3763737712597d06/djangoproject/urls/www.py), for example).

Reply:
ok